target_stitch/__init__.py

```diff
@@ -736,15 +744,15 @@ def exception_is_4xx(ex):
                       giveup=exception_is_4xx,
                       on_backoff=_log_backoff)
 async def post_coroutine(url, headers, data, verify_ssl):
-    # LOGGER.info("POST starting: %s ssl(%s)", url, verify_ssl)
+    LOGGER.info("POST starting: %s ssl(%s)", url, verify_ssl)
```

Comment:
Adding this back will lead to a large amount (ie too many) log messages, correct?

Reply:
good catch